core/src/main/java/org/apache/accumulo/core/client/admin/ActiveCompaction.java

```diff
@@ -145,6 +145,8 @@ enum Type {
     String getAddress();
 
     int getPort();
+
+    String getResourceGroup();
```

Comment:
We should open a follow on issue to look into deprecating this CompactionHost interface in favor of the new ServerId type.

Reply:
Commented on #4849 regarding this.